stubs/redis/redis/commands/core.pyi

```diff
@@ -112,10 +112,10 @@ class ManagementCommands:
     def memory_malloc_stats(self, **kwargs): ...
     def memory_usage(self, key, samples: Any | None = ..., **kwargs): ...
     def memory_purge(self, **kwargs): ...
-    def ping(self, **kwargs): ...
+    def ping(self, **kwargs) -> bool: ...
     def quit(self, **kwargs): ...
     def replicaof(self, *args, **kwargs): ...
-    def save(self, **kwargs): ...
+    def save(self, **kwargs) -> bool: ...
```

Comment:
If I read the documentation correctly, this should return something like `"OK"` on success. https://redis.io/commands/save